test/e2e/rule_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"time"
 
 	"github.com/efficientgo/e2e"
+	"github.com/pkg/errors"
```

Comment:
I was surprised to see this :smile: shouldn't we prefer `github.com/thanos-io/thanos/pkg/errors` now? :thinking: I see that `faillint` still has `errors=github.com/pkg/errors` so maybe I'm wrong. Sorry for noticing at first

Reply:
It's used in a lot of places, but yes we need to migrate off of it to `github.com/thanos-io/thanos/pkg/errors`! 🙂

Reply:
+1 to @GiedriusS. Would be great to not add one more thing that has to be fixed soon. Also `pkg/errors` doesn't guarantee compatibility with the future "go2 errors". `ithub.com/thanos-io/thanos/pkg/errors` uses the stdlib `errors`, which should keep us compatible with future releases.

Reply:
Done in https://github.com/thanos-io/thanos/pull/5558/commits/914aaae683dc86716a019289dd609034c39af6b8